File: Java/Java - Easy/Part - 004/Average Speed.java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
		//Your Code Here
		Scanner sc = new Scanner(System.in);
		String[] input = sc.nextLine().split(" ");
		int distance = 0, time = 0;
		for(String data: input){
		    String[] parsedData = data.split("@");
		    distance += Integer.parseInt(parsedData[0]);
		    time += Integer.parseInt(parsedData[1]);
		}
		System.out.format("%.2f kmph",(double) distance / (double) time);

	}
}

File: Python/Python - EASY/PART - 004/Average Speed.py
values = input().split()

distance = []
time = []

for value in values:
    temp = value.split("@")
    distance.append(int(temp[0]))
    time.append(int(temp[1]))

result = sum(distance) / sum(time)
print("{:.2f} kmph".format(result))
